department-of-veterans-affairs/abd-vro
# Add flag in API response to monitor dropdown selection

## What was the problem?

These changes modify the API response to check if the contention_text was selected from the dropdown list. This will allow the CC team to monitor the use of the dropdown options. 

Associated tickets or Slack threads:
- [#CC-283](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/283)

## How does this fix it?[^1]


## How to test this PR
- From domain-cc/cc-app run `pytest`
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
